### PR TITLE
address the remaining package:lints issues

### PR DESCRIPTION
**Pull request by devoncarew**
- rename files
- avoid empty try blocks
- fix several unrelated_type_equality_checks issues
- various misc updates to the readme

### CODE REVIEW

**Comment by devoncarew** [comment]
@pq @csells - ping for review; thanks!

File: bin/enumerate_team_members.dart [hunk 1]
```diff
@@ -124,13 +124,17 @@ void main(List<String> args) async {
     dynamic earliestAuthored;
     try {
       earliestAuthored = await github.searchIssuePRs(earliestQueryAuthor).first;
-    } catch (e) {}
+    } catch (e) {
```
Comment (pq) [inline bin/enumerate_team_members.dart]: Since `e` is ignored, consider renaming it `_`.

File: bin/enumerate_team_members.dart [hunk 2]
```diff
@@ -124,13 +124,17 @@ void main(List<String> args) async {
...
+      print('Unable to retrieve earliestQueryAuthor.');
+    }
 
     dynamic earliestCommented;
     try {
       earliestCommented =
           await github.searchIssuePRs(earliestQueryCommenter).first;
-    } catch (e) {}
+    } catch (e) {
```
Comment (pq) [inline bin/enumerate_team_members.dart]: And here and below.

**Review by pq — APPROVED** [review]

File: bin/enumerate_team_members.dart (hunk 1 above)
Comment (devoncarew) [reply bin/enumerate_team_members.dart]: Thanks! Updated.

File: bin/enumerate_team_members.dart (hunk 2 above)
Comment (devoncarew) [reply bin/enumerate_team_members.dart]: done

**Pull request merged**